src/main/java/com/playmonumenta/structures/commands/ForceConquerRespawn.java

```diff
@@ -0,0 +1,49 @@
+package com.playmonumenta.structures.commands;
+
+import com.playmonumenta.structures.managers.RespawnManager;
+import dev.jorel.commandapi.CommandAPICommand;
+import dev.jorel.commandapi.CommandPermission;
+import dev.jorel.commandapi.arguments.PlayerArgument;
+import dev.jorel.commandapi.arguments.StringArgument;
+import org.bukkit.ChatColor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+public class ForceConquerRespawn {
+	public static void register() {
+		final String command = "forceconquerrespawn";
+		final CommandPermission perms = CommandPermission.fromString("monumenta.structures.forceconquerrespawn");
+
+		new CommandAPICommand(command)
+			.withPermission(perms)
+			.withArguments(new StringArgument("label").overrideSuggestions((sender) -> {return RespawnManager.getInstance().listStructures();}))
+			.executes((sender, args) -> {
+				if (sender instanceof Player) {
+					forceRespawn(sender, (String)args[0]);
+				}
+			})
```

Comment:
You can change this to ```.executesPlayer((player, args) -> {``` to only be valid when run by players, and eliminate the need to cast

Reply:
Wouldn't that not work because sometimes it's getting run by a function but passing a player in as an arg?

Reply:
I'm just saying change this one, not the other one too. The other one yeah will have non-player senders, but you've already set this one up so it does nothing if not run by a player

Reply:
Gotcha, fixing now